lib/searchable_by/concern.rb

```diff
@@ -26,20 +26,17 @@ def search_by(query)
       columns = config.columns
       return all if columns.empty?
 
-      values = Util.norm_values(query, min_length: config.min_length).first(config.max_terms)
```

Comment:
yeah, I am not a fan of this, I don't want to parse each value by tokenizer. what you do below is a nice idea, but it won't work well in practice and creates quite a bit of overhead :(. I will close this, we need to figure out a different way, let's discuss on slack